app/decorators/middleware_deployment_decorator.rb

```diff
@@ -3,11 +3,11 @@ class MiddlewareDeploymentDecorator < Draper::Decorator
 
   def fonticon
     if name.end_with? '.ear'
-      'product-file-ear-o'
+      'product product-file-ear-o'
     elsif name.end_with? '.war'
-      'product-file-war-o'
+      'product product-file-war-o'
     else
-      'product-report'
+      'fa fa-file-text-o'
```

Comment:
Well, there's 2 places mentioned in https://github.com/ManageIQ/manageiq/pull/9351, but probably there will be more (as in, any gtl screen showing this...)